packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -0,0 +1,136 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnIconTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { iconXmarkTag } from '../../../icons/xmark';
+import { tableColumnTextTag } from '../../text';
+import { iconCheckLargeTag } from '../../../icons/check-large';
+import { mappingIconTag } from '../../../mapping/icon';
+import { mappingSpinnerTag } from '../../../mapping/spinner';
+import { sharedMappingValidityDescription } from '../../enum-base/tests/shared-storybook-docs';
+import { isChromatic } from '../../../utilities/tests/isChromatic';
+
+const simpleData = [
+    {
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        status: 'fail',
+        isChild: true
+    },
+    {
+        firstName: 'Marge',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        status: 'calculating',
+        isChild: false
+    },
+    {
+        firstName: 'Bart',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: true
+    }
+];
+
+const iconColumnDescription = 'The `nimble-table-column-icon` column renders string, number, or boolean values as a Nimble icon or `nimble-spinner` in the `nimble-table`.';
```

Comment:
@jattasNI had made this comment on my number-text PR, and I think it's relevant here too. We should make a statement of what the sort/group behavior is. Specifically in this case, it would be good to note that the raw values are sorted & grouped, not the mapped values. It would probably be good to go back and add that same note to the enum-text column, possibly in a separate PR.

Reply:
I went ahead and updated the description for both elements.